app/controllers/application_controller.rb

```diff
@@ -1,15 +1,8 @@
 class ApplicationController < ActionController::API
   include ActionController::HttpAuthentication::Token::ControllerMethods
 
-  before_action :authenticate
```

Comment:
@misaka this might be a bit OTT defensive, but is it worth:

1. keeping the `before_action :authenticate`
2. have `authenticate` in this class throw an exception

The reason for that is:

* all endpoints in this app should and are behind authentication
* if someone inadvertently subclasses `ApplicationController`, rather than the `V*ApplicationController` it will be open by default

Reply:
hmmm literal leave this line in

`before_action :authenticate`

and dont have a `def authenticate` ?

Reply:
see https://github.com/DFE-Digital/manage-courses-backend/pull/110/commits/ac25d5dde884d78009e9fa2bb5dcc08957f8bb35